lib/ethpy/ethpy/base/transactions.py

```diff
@@ -219,6 +221,53 @@ def retry_preview_check(exc: Exception) -> bool:
     return {f"value{idx}": value for idx, value in enumerate(return_values)}
 
 
+def wait_for_transaction_receipt(
+    web3: Web3, transaction_hash: HexBytes, timeout: float = 30, start_latency: float = 1, backoff: float = 2
+) -> TxReceipt:
+    """wait_for_transaction_receipt with exponential backoff
```

Comment:
can you make this `web3.wait_for_transaction_receipt`

Reply:
web3's version doesn't have exp backoff. I'm using that here.